controlpanel/frontend/consumers.py

```diff
@@ -166,6 +168,23 @@ def get_tool_and_user(self, message):
         user = User.objects.get(auth0_id=message["user_id"])
         return tool, user
 
+    def home_reset(self, message):
+        """
+        Reset the home directory of the specified user.
+        """
+        user = User.objects.get(auth0_id=message["user_id"])
+        home_directory = HomeDirectory(user)
+        update_home_status(home_directory, HOME_RESETTING)
+        
+        home_directory.reset()
+
+        status = wait_for_home_reset(home_directory)
+
+        if status == HOME_RESET_FAILED:
+            log.error(f"Failed to reset home directory for user {user}")
+        else:
+            log.debug(f"Reset home directory for user {user}")
+
```

Comment:
Just a question: We send the status update but the UI doesn't use it right? I guess it's still good to send this updates even if the UI at the moment doesn't show any progress or errors (and it may be working and overkill to do something more clever I guess)

Reply:
Correct. It's there but not used. I was copying the code pattern I found already for background tasks and wanted to make it "complete". ;-)